plugins/CoreAdminHome/Commands/SetConfig.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\CoreAdminHome\Commands;
+
+use Piwik\Config;
+use Piwik\Plugin\ConsoleCommand;
+use Piwik\Plugins\CoreAdminHome\Commands\SetConfig\ConfigSettingManipulation;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class SetConfig extends ConsoleCommand
+{
+    protected function configure()
+    {
+        $this->setName('core:set-config');
```

Comment:
We should really avoid `core`. We already fixed this for some like `core:clear-caches` and renamed it to `cache:clear`, same for `plugin:...`. Apart from us `core` developer nobody really knows what we mean by `core` and it doesn't provide any real value to use this term @mattab